masterlist.yaml

```
@@ -15732,6 +15741,15 @@ plugins:
         itm: 18
         udr: 1
 
+  - name: 'StrosMKaiRum.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/43830/'
+        name: 'Unique Stros M''Kai Rum'
+    tag: [ Graphics ] 
```

Since the only non-`Graphics` change is an `Update.esm` forward, it can be tagged with `Deactivate` too. There's also an extra whitespace:

```suggestion
    tag:
      - Deactivate
      - Graphics
```